src/app/grievance-transaction-details/grievance-transaction-details.component.ts

```
@@ -0,0 +1,51 @@
+    this.assignSelectedLanguage();
+  }
+  assignSelectedLanguage() {
+    const getLanguageJson = new SetLanguageComponent(this.HttpServices);
+    getLanguageJson.setLanguage();
+    this.currentLanguageSet = getLanguageJson.currentLanguageObject;
+  }
+
+
+
+  viewTransactionFile(fileUrl:any) {
+      if (fileUrl && fileUrl.fileName) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)